stable/provazio/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.controller.enabled }}
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "provazio.controller.name" . }}
+  labels:
+    app: {{ template "provazio.controller.name" . }}
```

Comment:
the app is `provazio.name`, please add a `component` part to the labels with fixed name controller.
Look how we do Spark labels

Reply:
Done